snow/consensus/snowball/tree_test.go

```diff
@@ -742,9 +684,10 @@ func TestSnowballFineGrained(t *testing.T) {
 
 	{
 		expected := `SB(Preference = 0, PreferenceStrength[0] = 1, PreferenceStrength[1] = 0, SF(Confidence = 1, Finalized = false, SL(Preference = 0))) Bit = 0
-    SB(Preference = 0, PreferenceStrength[0] = 1, PreferenceStrength[1] = 0, SF(Confidence = 1, Finalized = false, SL(Preference = 0))) Bit = 2
-        SB(PreferenceStrength = 1, SF(Confidence = 1, Finalized = true)) Bits = [3, 256)
-        SB(PreferenceStrength = 0, SF(Confidence = 0, Finalized = false)) Bits = [3, 256)
+    SB(PreferenceStrength = 1, SF(Confidence = 1, Finalized = false)) Bits = [1, 2)
```

Comment:
See graphs added in https://github.com/ava-labs/avalanchego/pull/2949/commits/762e2d832efdd7688894048b37de7d2ac6a81fdb for visualization of the required changes here

The name doesn't give clear indication of what it's trying to test, so I just made sure that it passes correctly after the changes.